### ISSUE TITLE
compile output enhancement

### ISSUE THREAD

**Issue by lpdunwell** [arduino/Arduino#8150]
make 'error' and 'warning' more visible by displaying them **bold** and **red**

**Comment by dsyleixa** [comment]
I have to disagree about red color at a black bacground, it's extremely bad visible then: bold light orange instead would be much better.
OTOH, on a white background bold red  is supposed to be fine though.
